user_notification/migrations/0002_auto_20201223_2252.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from django.db import migrations, models
+
```

Comment:
Is it autogenreated file? The comment of "auto generated" is missing

Reply:
fixed